### ISSUE TITLE
GroupedVerticalBarChart: Bar chart is not updated when the chart data is updated

### ISSUE THREAD

**Issue by luckyrass** [microsoft/fluentui#13838]
### Environment Information

- **Package version(s)**: (@uifabric/charting_v2.3.5)
- **Browser and OS versions**: (fill this out if relevant)

### Please provide a reproduction of the bug in a codepen:

**Description**
Click the "Change data" button to update the chart data of the second groupedVerticalBarChart. The X-axis and Y-axis is updated, but the bar chart is not updated.

#### Actual behavior:

![image](https://user-images.githubusercontent.com/12230173/85990780-1bbaba80-ba25-11ea-849a-3f72be239f1c.png)

#### Expected behavior:

![image](https://user-images.githubusercontent.com/12230173/85990656-edd57600-ba24-11ea-8767-718715ed114e.png)

**How to repo it?**
Update the example code in fluentui\packages\charting\src\components\GroupedVerticalBarChart\examples\GroupedVerticalBarChart.Basic.Example.tsx
and click the Change data button.

```
import * as React from 'react';
import {
  GroupedVerticalBarChart,
  IGroupedVerticalBarChartProps,
  IGroupedVerticalBarChartData,
} from '@uifabric/charting';
import { DefaultPalette } from 'office-ui-fabric-react/lib/Styling';
import { mergeStyles } from 'office-ui-fabric-react/lib/Styling';

import { DefaultButton } from 'office-ui-fabric-react/lib/Button';

export interface IGroupState {
  data: IGroupedVerticalBarChartData[];
}

export class GroupedVerticalBarChartBasicExample extends React.Component<IGroupedVerticalBarChartProps, IGroupState> {
  private _data = [
    {
      name: '2000',
      series: [
        {
          key: 'series1',
          data: 90000,
          color: DefaultPalette.accent,
          legend: 'MetaData1',
          xAxisCalloutData: '2020/04/30',
          yAxisCalloutData: '9%',
        },
        {
          key: 'series2',
          data: 85000,
          color: DefaultPalette.blueMid,
          legend: 'MetaData2',
          xAxisCalloutData: '2020/04/30',
          yAxisCalloutData: '85%',
        },
        {
          key: 'series3',
          data: 36000,
          color: DefaultPalette.blueLight,
          legend: 'MetaData3',
          xAxisCalloutData: '2020/04/30',
          yAxisCalloutData: '36%',
        },
        {
          key: 'series4',
          data: 66000,
          color: DefaultPalette.blue,
          legend: 'MetaData4',
          xAxisCalloutData: '2020/04/30',
          yAxisCalloutData: '66%',
        },
        {
          key: 'series5',
          data: 34000,
          color: DefaultPalette.blueDark,
          legend: 'MetaData5',
          xAxisCalloutData: '2020/04/30',
          yAxisCalloutData: '34%',
        },
      ],
    },
    {
      name: '2010',
      series: [
        {
          key: 'series1',
          data: 14000,
          color: DefaultPalette.accent,
          legend: 'MetaData1',
          xAxisCalloutData: '2020/04/30',
          yAxisCalloutData: '14%',
        },
        {
          key: 'series2',
          data: 50000,
          color: DefaultPalette.blueMid,
          legend: 'MetaData2',
          xAxisCalloutData: '2020/04/30',
          yAxisCalloutData: '50%',
        },
        {
          key: 'series3',
          data: 33000,
          color: DefaultPalette.blueLight,
          legend: 'MetaData3',
          xAxisCalloutData: '2020/04/30',
          yAxisCalloutData: '33%',
        },
        {
          key: 'series4',
          data: 44000,
          color: DefaultPalette.blue,
          legend: 'MetaData4',
          xAxisCalloutData: '2020/04/30',
          yAxisCalloutData: '44%',
        },
        {
          key: 'series5',
          data: 72000,
          color: DefaultPalette.blueDark,
          legend: 'MetaData5',
          xAxisCalloutData: '2020/04/30',
          yAxisCalloutData: '72%',
        },
      ],
    },
    {
      name: '2020',
      series: [
        {
          key: 'series1',
          data: 33000,
          color: DefaultPalette.accent,
          legend: 'MetaData1',
          xAxisCalloutData: '2020/04/30',
          yAxisCalloutData: '33%',
        },
        {
          key: 'series2',
          data: 3000,
          color: DefaultPalette.blueMid,
          legend: 'MetaData2',
          xAxisCalloutData: '2020/04/30',
          yAxisCalloutData: '3%',
        },
        {
          key: 'series3',
          data: 75000,
          color: DefaultPalette.blueLight,
          legend: 'MetaData3',
          xAxisCalloutData: '2020/04/30',
          yAxisCalloutData: '75%',
        },
        {
          key: 'series4',
          data: 29000,
          color: DefaultPalette.blue,
          legend: 'MetaData4',
          xAxisCalloutData: '2020/04/30',
          yAxisCalloutData: '29%',
        },
        {
          key: 'series5',
          data: 50000,
          color: DefaultPalette.blueDark,
          legend: 'MetaData5',
          xAxisCalloutData: '2020/04/30',
          yAxisCalloutData: '90%',
        },
      ],
    },
  ];

  private _data1 = [
    {
      name: 'today',
      series: [
        {
          key: 'series1',
          data: 100,
          color: 'red',
          legend: 'MetaData1',
          xAxisCalloutData: '2020/04/30',
          yAxisCalloutData: '9%',
        },
        {
          key: 'series2',
          data: 200,
          color: DefaultPalette.blueMid,
          legend: 'MetaData2',
          xAxisCalloutData: '2020/04/30',
          yAxisCalloutData: '85%',
        },
      ],
    },
    {
      name: 'tomorrow',
      series: [
        {
          key: 'series1',
          data: 300,
          color: 'red',
          legend: 'MetaData1',
          xAxisCalloutData: '2020/04/30',
          yAxisCalloutData: '14%',
        },
        {
          key: 'series2',
          data: 400,
          color: DefaultPalette.blueMid,
          legend: 'MetaData2',
          xAxisCalloutData: '2020/04/30',
          yAxisCalloutData: '50%',
        },
      ],
    },
  ];

  constructor(props: IGroupedVerticalBarChartProps) {
    super(props);
    this.state = {
      data: this._data,
    };

    this._changeData = this._changeData.bind(this);
  }

  public render(): React.ReactNode {
    const rootStyle = mergeStyles({ width: '650px', height: '400px' });

    return (
      <div className={rootStyle}>
        <GroupedVerticalBarChart data={this._data} height={400} width={650} showYAxisGridLines />
        <GroupedVerticalBarChart data={this.state.data} height={400} width={650} showYAxisGridLines />
        <DefaultButton text="Change data" onClick={this._changeData} />
      </div>
    );
  }

  private _changeData(): void {
    this.setState({
      data: this._data1,
    });
  }
}
```

### Priorities and help requested:

Are you willing to submit a PR to fix? (Yes, No)

Requested priority: (Blocking)

Products/sites affected: (if applicable)

**Comment by mnajdova** [comment]
@Raghurk is this a known issue?

**Comment by Raghurk** [comment]
We are investigating. Thanks!

**Comment by jameelakowsar** [comment]
#PRRaised
https://github.com/microsoft/fluentui/pull/13853

**Comment by jameelakowsar** [comment]
updated code available in @uifabric/charting@v2.3.8

**Comment by mnajdova** [comment]
Thanks for fixing this @jameelakowsar Closing it
